### PR TITLE
Bumping OLM to 1.10

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Me naive trying to just bump the OLM version to 1.10 ... :grin:

### CODE REVIEW

**Comment by jcrossley3** [comment]
I don't know if this is correct, but it is nice that a bump is now so small due to the new structure. I hated having to create entirely new (and mostly redundant) CSV files for each version bump.

**Comment by matzew** [comment]
@warrenvw  this here: https://github.com/matzew/serverless-operator/blob/bump_olm_to_1.10/hack/lib/catalogsource.bash#L82-L83  (on muaster and 1.9.0) still uses 1.7.2 / 1.8.0 :thinking: 

should that be 1.8.0 / 1.9.0 instead ?

**Comment by matzew** [comment]
I am currently not exactly sure, how to handle this error message:

```
time="2020-09-10T14:55:33Z" level=error msg="permissive mode disabled" bundles="[image-registry.openshift-image-registry.svc:5000/openshift-marketplace/serverless-bundle]" error="Invalid bundle serverless-operator.v1.9.0, bundle specifies a non-existent replacement serverless-operator.v1.9.0"
Error: Invalid bundle serverless-operator.v1.9.0, bundle specifies a non-existent replacement serverless-operator.v1.9.0
Usage:
```

/cc @warrenvw

**Comment by matzew** [comment]
we dont have an 1.9.0 image - at tis point

File: hack/lib/catalogsource.bash [hunk 1]
```diff
@@ -80,7 +80,7 @@ spec:
         - |-
           podman login -u $pull_user -p $token image-registry.openshift-image-registry.svc:5000 && \
           /bin/opm registry add                         -d index.db --mode=replaces -b docker.io/markusthoemmes/serverless-bundle:1.7.2 && \
```
Comment (matzew) [inline hack/lib/catalogsource.bash:82]: @markusthoemmes or @warrenvw 

Do we have a new/different image?  Is there a repo? Curious, b/c of "markusthoemmes" docker.io account :smile_cat:

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -33,8 +33,8 @@ metadata:
       Provides a collection of API's based on Knative to support deploying and serving
       of serverless applications and functions.
     repository: https://github.com/openshift-knative/serverless-operator
-    support: Red Hat
-  name: serverless-operator.v1.9.0
+    support: Red Hat, Inc.
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:36]: Please drop this again.

File: hack/lib/catalogsource.bash (hunk 1 above)
Comment (markusthoemmes) [reply hack/lib/catalogsource.bash:82]: I guess @warrenvw's work in https://github.com/openshift-knative/serverless-operator/pull/414 would replace this.

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:36]: Tattoo INC ! :-)

**Pull request merged**